cli/cmd/install.go

```diff
@@ -591,7 +592,7 @@ func (options *installOptions) globalConfig(identity *pb.IdentityContext) *pb.Gl
 		LinkerdNamespace:  controlPlaneNamespace,
 		AutoInjectContext: autoInjectContext,
 		CniEnabled:        options.noInitContainer,
-		Version:           options.linkerdVersion,
+		Version:           options.proxyVersion,
```

Comment:
This isn't correct. the global version should be the installed control plane version. It sounds like we may need the ProxyConfig to get a Version field...